packages/components/src/focal-point-picker/styles/focal-point-picker-style.ts

```diff
@@ -53,7 +53,7 @@ export const MediaPlaceholder = styled.div`
 `;
 
 export const StyledUnitControl = styled( UnitControl )`
-	width: 100px;
+	width: 100%;
```

Comment:
Noting that this change affects the component regardless of the `__next40pxDefaultSize` prop (which is not necessarily an issue, but wanted to make sure we're aware of it)

Reply:
I'll let @richtabor clarify if needed, but I believe the change was made to fill the gap/make the spacing more consistent. Since the inputs are wrapped in flex, I don't think it would cause any issues. 

| Before  | After |
| ------------- | ------------- |
| <img width="296" alt="Screenshot 2023-11-23 at 11 50 05 AM" src="https://github.com/WordPress/gutenberg/assets/35543432/11f88c74-efcd-400f-8aea-47f648f05540">  | <img width="282" alt="Screenshot 2023-11-23 at 11 50 13 AM" src="https://github.com/WordPress/gutenberg/assets/35543432/cb49e730-b4e5-4f31-ba20-d0fb85332b0e">  |

Reply:
Yes, exactly; filling the space is ideal.